app.js

```diff
@@ -0,0 +1,41 @@
+// const input = document.getElementById('number-input');
+const submitButton = document.getElementById('submit-button');
+const guess = document.getElementById('submitted-guess');
+// const result = document.getElementById('submitted-result');
+const wonOrLost = document.getElementById('won-or-lost');
+const guessResult = document.getElementById('guess-result');
+// const guessesToGo = document.getElementById('guesses-to-go');
+// const outcomeResponse = document.getElementById('outcome-response');
+// const resetButton = document.getElementById('reset');
+// const mainArea = document.getElementById('main-area');
+
+const generatedNumber = Math.floor(Math.random() * 20 + 1) ;
+
+const decision = (guess - generatedNumber);
+
+// function beginGame() {
+//     let correctAnswers = 0;
+//     let remainingGuesses = 4;
+// }
+
+// function winner() {
+//     wonOrLost.textContent = 'You Are a Winner!';
```

Comment:
this is a perfect line, but it does not have to be inside of a function to work properly.  You would want to execute this line inside of some logic that takes the output of compareNumbers and then decides which textcontent to use based on the results of that function.  ie.  the output is 0,  so your text content could be "you won!"